README.md

```diff
@@ -17,7 +17,7 @@
 </h3>
 
 <p align="center">
```

Comment:
Can you also please remove the empty tag

Reply:
Done